.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
Ah, I see you don't have a package.json in the repo yet. You should probably set one up to set the dependencies you need for the site :+1: